<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>S1_frame_merge</class>
 <widget class="QWidget" name="S1_frame_merge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>哨兵一号相邻frame拼接</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../QtWidgetsApplication3.qrc">
    <normaloff>:/QtWidgetsApplication3/bin/icon/frame_merge.png</normaloff>:/QtWidgetsApplication3/bin/icon/frame_merge.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>干涉相位拼接</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>SAR单视复图像拼接</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,5">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>选择工程</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_project"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1,6">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>选择节点</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_node1"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>选择数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_data1"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,1,6">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>选择节点</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_node2"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>选择数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_data2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="5,5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>目标节点名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dstnode"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../QtWidgetsApplication3.qrc"/>
 </resources>
 <connections/>
</ui>
